health/service/src/main/java/com/cerner/beadledom/health/HealthDependency.java

```diff
@@ -9,7 +9,22 @@
  */
 public abstract class HealthDependency {
 
-  private Boolean primary = false;
+  private Boolean primary;
+
+  /**
+   * A constructor for Heath Dependencies that sets the dependency to be a primary one.
+   */
+  public HealthDependency() {
+    primary = true;
+  }
+
+  /**
+   * A constructor that allows the user to pass in if the dependency is primary or not.
+   * @param primary A boolean to indicate if the health dependency is primary or not.
```

Comment:
Add a line break before the at-params. 

The at-params should *not* be full sentences, so start with a lower case letter and do not include a period at the end.

Reply:
Added the line break and made it not a sentence: https://github.com/cerner/beadledom/pull/123/commits/95686b23b05f48a497bd6fac0a15acfb9712165b